### PR TITLE
EUI-8961 ROO Companies house page corrections

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/EUI-8961

### Change description ###
Register other org - Correction to Companies House number error message display and change 'company' to 'organisation' 

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by udaydenduluri33 — APPROVED** [review]
LGTM

File: src/register-org/components/company-house-details/company-house-details.component.html [hunk 1]
```diff
@@ -28,10 +28,10 @@ <h1 class="govuk-heading-xl">What is your company name and Companies House numbe
             Enter the 8-digit Companies House Number (optional)
           </label>
           <div id="company-house-number-hint" class="govuk-hint">
-            This helps HMCTS to verify your company on our System
+            This helps HMCTS to verify your organisation on our System
```
Comment (Daniel-Lam) [inline src/register-org/components/company-house-details/company-house-details.component.html:31]: @johnbenjamin-hmcts - not related to your changes, but I'm not sure why we're capitalising the 's' in "System"... I've seen this elsewhere and would suggest it's just "system"

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - one very minor suggestion but otherwise it's all good

File: src/register-org/components/company-house-details/company-house-details.component.html (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/register-org/components/company-house-details/company-house-details.component.html]: @Daniel-Lam - I have updated the code now. Could you please re-review?

Comment (Daniel-Lam) [reply src/register-org/components/company-house-details/company-house-details.component.html]: @johnbenjamin-hmcts - all good

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**